trading_report/src/main.ts

```diff
@@ -48,7 +49,8 @@ async function main() {
   const currentTime = (new Date().getTime() / 1_000);
   const cutoffTime = currentTime - 24 * 60 * 60;
 
-  const opbUrl = `https://mango-transaction-log.herokuapp.com/v4/stats/openbook-trades?address=${openOrdersAccount.toBase58()}&address-type=open-orders&limit=10000`
+  // Assumes that all trades happen on the first open orders account.
```

Comment:
Why is this assumption safe?

Reply:
Might not be. But in practice, we only have it for SOL and it works there.